group_vars/location_segen/networks.yml

```diff
@@ -0,0 +1,188 @@
+---
+# IPv4 10.31.6.0/24
```

Comment:
Why two /24? Please stop wasting IPs

Reply:
These IP's, as stated above, have belonged to Segen since day 1.

Reply:
As discussed with all participants: Both /24 IP-Blocks should be kept for now as they were already assigned. This might not be optimal in terms of IPv4 address usage but this should not be a blocker for finalizing and merging this. We can reevaluate dropping the additional block after Corona, once we have more statistics based on the next "Hoffest".